hathor/cli/run_node.py

```diff
@@ -80,6 +80,8 @@ def create_parser(self) -> ArgumentParser:
         parser.add_argument('--max-output-script-size', type=int, default=None, help='Custom max accepted script size '
                             'on /push-tx API')
         parser.add_argument('--sentry-dsn', help='Sentry DSN')
+        parser.add_argument('--x-sync-bridge', action='store_true',
```

Comment:
Should we add it to the list of unsafe parameters for now?

Reply:
I think we should, will do when that is merged.